### DISCUSSION TITLE
Textarea to bbcode (with new lines)

### DISCUSSION THREAD

**Discussion by notrab** [JiLiZART/BBob#72] [Q&A]
Hey! Great library!

I am a long time BBCode fan, and wanting to do some livestreams integrating bbob into a project I'm working on.

One question is regarding the best practices on storing the BBcode data.

Is it preferred to store the AST, or plain text?

If the AST is required to be saved to the DB, what is the best function/helper to use here? One other thing would be storing, and parsing new lines. What is your recommend here?

I'm using React. I'm happy to contribute to docs once I know the answer to make it easier for others.

**Comment by JiLiZART** [comment]
Hi :) I prefer store plain text, cuz it's more safe, you can change your bb code parsing infrastructure easily.
What about new lines, you can create plugin to handle this (if u need specific behavior) for ex https://codesandbox.io/s/youthful-bose-2kjsq?file=/src/index.js

**Comment by notrab** [comment]
Hi @JiLiZART 

Thanks for responding so quickly. I was also in favour of storing plain text.

I tried viewing the CSB but the link just shows a CRA boilerplate.

**Comment by JiLiZART** [comment]
F8k, i forgot to save :(
